provisioners/group_vars/mesos

```diff
@@ -0,0 +1,45 @@
+---
+# Note: Try to minimize the number of group-specific variables
+
+use_sudo: no
+#TODO add apache module to local/all group_vars
+ace_force_reinstall: False
+
+apache_as_service: False
+apache_config_dir: "{{lookup('env', 'MESOS_SANDBOX')}}/apache_ckan.conf"
+apache_server_root: "{{lookup('env', 'MESOS_SANDBOX')}}/httpd"
+apache_port: 8888
```

Comment:
This should probably be {{ lookup('env', 'PORT0') }}, to pull in the port from Marathon, no?